From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sun, 11 Oct 2015 20:27:28 +0200
Subject: ar71xx: remove WZR-600DHP from Buffalo WZR-HP-AG300H/WZR-600DHP model name string for now

We can't use the same image for these two devices, so as a workaround,
remove ZR-600DHP from the name for now, so the autoupdater can work and
users aren't confused.

diff --git a/target/linux/ar71xx/base-files/lib/ar71xx.sh b/target/linux/ar71xx/base-files/lib/ar71xx.sh
index e1d03ea..1e50b5f 100755
--- a/target/linux/ar71xx/base-files/lib/ar71xx.sh
+++ b/target/linux/ar71xx/base-files/lib/ar71xx.sh
@@ -886,7 +886,7 @@ ar71xx_board_detect() {
 	*"WZR-450HP2")
 		name="wzr-450hp2"
 		;;
-	*"WZR-HP-AG300H/WZR-600DHP")
+	*"WZR-HP-AG300H")
 		name="wzr-hp-ag300h"
 		;;
 	*WZR-HP-G300NH)
diff --git a/target/linux/ar71xx/files/arch/mips/ath79/mach-wzr-hp-ag300h.c b/target/linux/ar71xx/files/arch/mips/ath79/mach-wzr-hp-ag300h.c
index edd48f2..a016a3d 100644
--- a/target/linux/ar71xx/files/arch/mips/ath79/mach-wzr-hp-ag300h.c
+++ b/target/linux/ar71xx/files/arch/mips/ath79/mach-wzr-hp-ag300h.c
@@ -202,4 +202,4 @@ static void __init wzrhpag300h_setup(void)
 }
 
 MIPS_MACHINE(ATH79_MACH_WZR_HP_AG300H, "WZR-HP-AG300H",
-	     "Buffalo WZR-HP-AG300H/WZR-600DHP", wzrhpag300h_setup);
+	     "Buffalo WZR-HP-AG300H", wzrhpag300h_setup);
